tests/init.go

```diff
@@ -98,14 +98,15 @@ var Forks = map[string]*params.ChainConfig{
 		EIP158Block:    big.NewInt(0),
 		ByzantiumBlock: big.NewInt(5),
 	},
-	"ByzantiumToConstantinopleAt5": {
+	"ByzantiumToConstantinopleFixAt5": {
 		ChainID:             big.NewInt(1),
 		HomesteadBlock:      big.NewInt(0),
 		EIP150Block:         big.NewInt(0),
 		EIP155Block:         big.NewInt(0),
 		EIP158Block:         big.NewInt(0),
 		ByzantiumBlock:      big.NewInt(0),
-		ConstantinopleBlock: big.NewInt(5),
+		ConstantinopleBlock: big.NewInt(0),
```

Comment:
Shouldn't Constantinople also be at block 5?

Reply:
Yeah, as it was, it was correctly byzantium to contantinople at `5`. With this change, it becomes Constantinople to ConstantinopleFix at `5`.

Reply:
This is question to @winsvega. It was configured this way in Aleth.

Reply:
https://github.com/ethereum/aleth/commit/6739ab44a3290bf87abf65119fa3b9efe365892d#diff-f632409b732e65eaca1150537ec753e8L29

Reply:
Yes you right. if it statest Byzantium to ConstantinopleFix At 5  
Constantinople (with sstore change) must be enabled at block5
Then at the same time at block5 ConstantinopleFix is enabled (disbaling the sstore change) 
If it is not like this in aleth it is probably a typo. So that should trigger an error for othe clients on hive.

Reply:
@winsvega Are you planning to do any changes to tests related to this? There are 2 day left to the HF.